src/app/components/window-terminal/window-terminal.component.ts

```diff
@@ -52,6 +52,19 @@ export class WindowTerminalComponent implements OnInit {
     }, false);
   }
 
+  @HostListener('click', ['$event']) handleURLClick(event) {
```

Comment:
@jkuri Thank you for the review. I initially used `event: Mouseevent` but to avoid `Property tagName doesn't exist on type EventTarget` i changed it to Event. 
Is it fine if i leave those warnings like this? 
Or another workaround I was thinking to cast it inline like `(event.target as any).href` or `(event.target as any).tagName`. 
What would yo suggest